packages/cli/src/cli.ts

```diff
@@ -1,7 +1,7 @@
 import { build } from "gluegun";
 
 interface args {
-  [key: string]: any
+  [key: string]: unknown
```

Comment:
Going to be using `unknown` more ;) thanks for introducing me to this: https://devblogs.microsoft.com/typescript/announcing-typescript-3-0-rc-2/#the-unknown-type

Reply:
There is a recommended rule which won't allow explicit any anyways :smile: